.schema/config.schema.json

```
@@ -929,6 +929,17 @@
           "additionalProperties": {
             "type": "string"
           }
+        },
+        "allowed_remote_headers": {
```

I think this should have a different name to more clearly indicate what it is doing. What do you think about:

```suggestion
        "forward_response_headers_to_upstream": {
```